diff --git a/node_modules/@types/fetch-mock/index.d.ts b/node_modules/@types/fetch-mock/index.d.ts
index 5900068..1938035 100644
--- a/node_modules/@types/fetch-mock/index.d.ts
+++ b/node_modules/@types/fetch-mock/index.d.ts
@@ -204,6 +204,13 @@ declare namespace fetchMock {
          * @default true
          */
         includeContentLength?: boolean;
+
+        /**
+         * Match calls that only partially match a specified body json. Uses the
+         * is-subset library under the hood, which implements behaviour the same
+         * as jest’s .objectContaining() method.
+         */
+        matchPartialBody?: boolean;
     }
 
     interface MockCall extends Array<string | RequestInit | undefined> {
@@ -581,6 +588,13 @@ declare namespace fetchMock {
              */
             warnOnFallback?: boolean;
 
+            /**
+             * Match calls that only partially match a specified body json. Uses
+             * the is-subset library under the hood, which implements behaviour
+             * the same as jest’s .objectContaining() method.
+             */
+            matchPartialBody?: boolean;
+
             /**
              * Reference to the Promise constructor of a custom Promise
              * implementation.
